L75-core-remove-grpc-channel-ping

```diff
@@ -0,0 +1,37 @@
+Title
```

Comment:
Please set the title.

Also, please rename this file so that it has a ".md" extension.

Reply:
Done.